autopts/wid/ccp.py

```diff
@@ -477,20 +484,22 @@ def hdl_wid_20206(params: WIDParams):
     """
     stack = get_stack()
 
-    chars = stack.ccp.events[defs.BTP_CCP_EV_CHRC_HANDLES][0]
-    chrc_list = [f'{chrc:04X}' for chrc in chars]
+    if len(stack.ccp.events[defs.BTP_CCP_EV_CHRC_HANDLES]) > 0:
```

Comment:
This is fine, but one could also have consider a simpler change with early termination:

```
if len(stack.ccp.events[defs.BTP_CCP_EV_CHRC_HANDLES]) == 0:
        return False
```